presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -262,6 +262,14 @@ Property Name                                      Description
 
```

Comment:
Commit tittle is too long. It wraps in github.

Reply:
made it shorter